onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -210,11 +210,217 @@ Return Value:
     }
 }
 
+MLAS_FORCEINLINE
+MLAS_FLOAT32X4
+MlasDequantizeLinearVector(
+    MLAS_INT32X4 IntVector,
+    MLAS_FLOAT32X4 ScaleVector,
+    MLAS_INT32X4 ZeroPointVector
+    )
+{
+#if defined(MLAS_NEON64_INTRINSICS)
+    return MlasMultiplyFloat32x4(vcvtq_f32_s32(vsubq_s32(IntVector, ZeroPointVector)), ScaleVector);
```

Comment:
There is a MlasSubtractInt32x4 if you want to switch vsubq_s32/_mm_sub_epi32 to use that.

Reply:
Better to switch. Fixed.